sdk-rs/src/dlob.rs

```diff
@@ -112,6 +127,85 @@ impl DLOBClient {
 
         RxStream(rx)
     }
+
+    /// Subscribe to an orderbook via WebSocket.
+    /// You MUST have a Drift Client initialized to use this fn.
```

Comment:
my bad on this anti-pattern, will be fixed in this PR: https://github.com/circuit-research/protocol-v2/pull/5